specification/attributes/column_naming_convention.md

```diff
@@ -24,6 +24,7 @@ Naming convention for columns appearing in billing data.
   * Column IDs SHOULD NOT use acronyms.
   * Column IDs MUST be alphanumeric with no special characters.
   * Columns that have an ID and a Name MUST have the `Id` or `Name` suffix in the Column ID. Display Name for a Column MAY avoid the Name suffix if there are no other columns with the same name prefix.
+  * Columns names MUST must not exceed a certain string length. Maximum 64 characters.
```

Comment:
@SonalGarg-SG Can you explain what the reasoning for needing a limit? That can help lead the discussion on whether we should include this type of provision.

Reply:
@udam-f2  reason being some of the databases have a limit on the length of the column name and also for the readability, it's just a good practice.  While you don't really expect many to give a column name that is less readable but just because many other databases have no constraint, it's not an unseen scenario so adding a little standard may help. Thanks !